sematic/api/endpoints/request_parameters.py

```diff
@@ -31,17 +33,90 @@
 ]
 
 
+@dataclass
+class SearchRequestParameters:
+    limit: int
+    order: Callable[[Any], Any]
+    cursor: Optional[str]
+    group_by: Optional[sqlalchemy.Column]
+    filters: Optional[BooleanClauseList]
+    include_fields: Optional[List[str]]
+
+
+def get_garbage_filters(request_args: Dict[str, str], supported_filters: List[str]) -> Tuple[bool, List[str]]:
```

Comment:
The filters themselves are not garbage. :D
```suggestion
def get_gc_filters(request_args: Dict[str, str], supported_filters: List[str]) -> Tuple[bool, List[str]]:
```